dashboard/app/access.go

```diff
@@ -49,6 +49,22 @@ func checkAccessLevel(c context.Context, r *http.Request, level AccessLevel) err
 // AuthDomain is broken in AppEngine tests.
 var isBrokenAuthDomainInTest = false
 
+func emailInAuthDomains(email string, cfg *GlobalConfig) bool {
```

Comment:
The alternative may be some func emailInAuthDomains(email string, authDomain ...string).
You'll need 1 loop like:
```golang
for _, domain := authDomain... {
    if authDomain != "" && strings.HasSuffix(email, cfg.AuthDomain) {
        return true
    }
}
```

Reply:
This seems more idiomatic yep. I refactored the code that way :)